graspy/align/orthogonal_procrustes.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2020 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+from sklearn.utils import check_array
+from scipy.linalg import orthogonal_procrustes
+
+from .base import BaseAlign
+
+
+class OrthogonalProcrustes(BaseAlign):
```

Comment:
We should discuss whether to just call this Procrustes and have multiple "method" arguments if we wanna incorporate the scaling options

Reply:
im in progress on implementing other methods rn

Reply:
process of*

Reply:
i still sort of like OrthogonalProcrustes vs SeedlessProcrustes, but we shall see after im done w/ above

Reply:
as opposed to Procrustes vs SeedlessProcrustes

Reply:
`SeedlessProcrustes` is also "orthogonal", is part of my semantic nitpicking. And I doubt we wanna call it `SeedlessOrthogonalProcrustes`...

Reply:
though maybe makes sense to keep as `OrthogonalProcrustes` for consistency with the scipy function, but not if we are then going to include non-orthogonal transformations someday.

Reply:
this makes sense. i was debating between Procrustes and OrthogonalProcrustes, and went with the latter because "orthogonal Procrustes" is sort of a proper name of that [specific problem](https://en.wikipedia.org/wiki/Orthogonal_Procrustes_problem). I kind of thought of that similar to Ase and Lse both convert nodes to vectors, but Node2Vec is a proper name of a different method. and it felt like SeedlessProcrustes has sort of became the colloquial name for the other algorithm. i don't know, i'm all for semantic consistency, so i'm happy to reconsider this.

Reply:
we can very well have `DiagonalProcrustes` and/or `ScalingProcrustes` as separate things.